Packs/SentryWire/Integrations/SentryWire/SentryWire.py

```diff
@@ -0,0 +1,380 @@
+                readable_output += f"{key} : {value}\n"
+            readable_output = readable_output[:-1]
+            outputs["checkstatus"].append(response[i].get("checkstatus"))
+            outputs["getpcaps"].append(response[i].get("getpcaps"))
+            outputs["metadata"].append(response[i].get("metadata"))
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='SentryWire.Investigator.Search',
+        outputs_key_field='SearchID',
+        outputs=outputs
+    )
+
+
+def delete_search_command(client: Client, args: dict[str, Any]):
+    search_id = args.get("search_id")
+    response = client.delete_search(search_id=str(search_id))
+    response["SearchID"] = search_id
+    return CommandResults(
+        readable_output=f"{search_id} has been deleted!",
+        outputs_prefix='SentryWire.Investigator.Deleted',
+        outputs_key_field='SearchID',
+        outputs=response
+    )
+
+
+def download_pcap_command(client: Client, args: dict[str, Any]):
+    search_id = args.get('search_id')
+    node_name = args.get('node_name')
+    file_entry = fileResult(
+        filename=f'{search_id}.pcap',
+        data=client.download_pcap(search_id=str(search_id), node_name=str(node_name)).content
+    )
+    return file_entry
+
+
+def download_metadata_command(client: Client, args: dict[str, Any]):
+    search_id = args.get('search_id')
+    node_name = args.get('node_name')
+    file_entry = fileResult(
+        filename=f'{search_id}.zip',
+        data=client.download_metadata(search_id=str(search_id), node_name=str(node_name)).content
+    )
+    return file_entry
+
+
+def get_search_status_command(client: Client, args: dict[str, Any]):
+    search_id = args.get('search_id')
+    node_name = args.get('node_name')
+
+    # Status request
+    response = client.get_search_status(search_id=str(search_id), node_name=str(node_name))
+
+    # Add SearchID/NodeName
+    response["SearchID"] = search_id
+
+    # Remove redundant/superfluous data
+    response = remove_redundant(
+        ["SearchName",
+         "SearchKey",
+         "ID",
+         "CaseName",
+         "MasterToken",
+         "SearchPorts",
+         "SubmittedTime",
+         "MaxChunk",
+         "SearchType"],
+        response
+    )
+
+    # Parse status
+    if "SearchResult" in response:
+        response, readable_output = parse_search_status(response)
+    else:
+        results = response["SearchStatus"]
+        readable_output = f"Search status: {results}"
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='SentryWire.Investigator.Status',
+        outputs_key_field='SearchID',
+        outputs=response
+    )
+
+
+def get_server_status_command(client: Client):
+    response = client.get_server_status()
+    status = json.loads(response.get("ServerInfo")).get("Status")
+    response["NodeName"] = json.loads(response.get("ServerInfo")).get("NodeName")
+
```

Comment:
This code is not handled right in the case an empty response will return or ServerInfo will return None the integration will return an uninformative python error also if the  response.get("ServerInfo") will return None .get("Status") will return an uninformative error as well. 
I added some code suggestion.
```suggestion
    server_info = {}
    if response:
        server_info = json.loads(response.get("ServerInfo"))
    else:
        raise DemistoException("Failed to get server status from Sentywire")
    
    status = server_info.get("Status")
    node_name = server_info.get("NodeName")
    response["NodeName"] = node_name

```